build.sbt

```diff
@@ -1,38 +1,27 @@
-import play.Project._
-
 organization := "io.sphere.shop"
 
 name := "sphere-donut"
 
 version := "1.0-SNAPSHOT"
 
-playJavaSettings
-
 libraryDependencies ++= Seq(
   javaCore,
   javaJdbc,
-  "io.sphere" %% "sphere-play-sdk" % "0.67.0" withSources(),
+  javaWs,
   "io.sphere.sdk.jvm" % "sphere-models" % "1.0.0-M16" withSources(),
   "io.sphere.sdk.jvm" % "sphere-java-client-apache-async" % "1.0.0-M16" withSources(),
   "org.apache.httpcomponents" % "httpasyncclient" % "4.0.2",
   "io.sphere.sdk.jvm" % "sphere-convenience" % "1.0.0-M16" withSources(),
-  "io.sphere.sdk.jvm" %% "sphere-play-2_2-java-client" % "1.0.0-M16" withSources(),
+  //"io.sphere.sdk.jvm" %% "sphere-play-2_4-java-client" % "1.0.0-M16" withSources(),
   "com.novocode" % "junit-interface" % "0.11" % "test,it",
-  "org.assertj" % "assertj-core" % "2.0.0" % "test,it", // change to 3.0.0 with Java 8
+  "org.assertj" % "assertj-core" % "3.0.0" % "test,it",
```

Comment:
there is a newer version available: http://search.maven.org/#artifactdetails%7Corg.assertj%7Cassertj-core%7C3.1.0%7Cbundle